src/api/entitlements.js

```diff
@@ -178,6 +231,13 @@ export class Entitlements {
   ack() {
     this.ackHandler_(this);
   }
+
+  /**
+   * A 3p site should call this method to consume a Google metering entitlement.
```

Comment:
still not sure what it means to consume a GME

Reply:
I expanded this comment